tests/test_stream.py

```diff
@@ -76,5 +90,5 @@ def test_missing_index_json_local(local_remote_dir: Any):
     else:
         raise Exception(f"Missing {os.path.join(remote_dir, 'index.json')}")
     stream = Stream(remote=None, local=remote_dir)
-    with pytest.raises(RuntimeError, match='No `remote` provided, but local file.*'):
+    with pytest.raises(ValueError):
```

Comment:
Is there any specific reason for removing `match`? Valuing against the exception message would be good to ensure we capture the correct exception.

Reply:
Fixed